### ISSUE TITLE
Uncaught TypeError: Cannot read property 'click' of null

### ISSUE THREAD

**Issue by ghost** [LJeremy/instaccept#6]
I tried using the console method and got this.

```
VM204:18 Uncaught TypeError: Cannot read property 'click' of null
    at <anonymous>:18:116
(anonymous) @ VM204:18
setTimeout (async)
(anonymous) @ VM204:17
setTimeout (async)
(anonymous) @ VM204:15
```
![Screenshot (7)](https://user-images.githubusercontent.com/57326660/71426324-e304e780-2675-11ea-9736-9feb49d1e966.png)

![Screenshot (8)](https://user-images.githubusercontent.com/57326660/71426349-1b0c2a80-2676-11ea-8649-c473de596d60.png)

Thanks.

**Comment by andreich1980** [comment]
Duplicate of #1

**Comment by andreich1980** [comment]
Hey @LJeremy Seems that Instagram changed DOM structure. 
I could make a PR to fix it if you like.

**Comment by LJeremy** [comment]
@andreich1980 Sure, go ahead.

**Comment by andreich1980** [comment]
@LJeremy turned out it's a feature for closed profiles or something like that. I don't have one so I can't get the XPaths. Sorry :)

**Comment by LJeremy** [comment]
Closed, fixed by #11
